release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -934,6 +934,17 @@ BMER was deprecated in {product-title} version 4.15 and 4.16. With this release,
 [id="ocp-4-17-image-registry-bug-fixes_{context}"]
 ==== Image Registry
 
+* In {product-title} 4.14, installing a cluster with Azure AD Workload Identity was made generally available. With that feature, administrators can configure a Microsoft Azure cluster to use Azure AD Workload Identity. With Azure AD Workload Identity, cluster components use short-time security credentials that are managed outside of the cluster. 
++
+Before this update, when {product-title} was deployed on Azure clusters with Azure AD Workload Identity, storage accounts that were created for the cluster and the image registry had storage account key access enabled by default. The use of storage account keys like Shared Access Signatures (SAS) could create security risks for some {product-title} on Azure deployments.
++
+With this update, when cluster administrators create a new storage account, shared access keys are disabled by default for image registry storage accounts that use Azure AD Workload Identity. This change enhances security by preventing the use of shared access keys.
++
+Administrators should note that existing storage accounts created before this update do not automatically have shared access keys disabled. Administrators must manually disable access keys support on their storage accounts to prevent the use of shared keys. For more information, see. . .
```

Comment:
Here are the Azure docs that cover disabling shared access key: https://learn.microsoft.com/en-us/azure/storage/common/shared-key-authorization-prevent?tabs=portal
Maybe we can just have a link to that here?

Reply:
Added